xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml

```diff
@@ -42,12 +42,12 @@
   <content>invitation.applicationsPanelEntryLabel=Invitation
 
 # Invitation Administration Section
-Invitation.WebHome_from_address.hint=The email address used to send invitation emails from.
-Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server, only when authentication is required.
-Invitation.WebHome_smtp_server_username.hint=The user name to connect to the SMTP server, only when authentication is required.
+Invitation.WebHome_from_address.hint=The email address used to send the invitation emails.
```

Comment:
I think this loose a bit the meaning, we don't know anymore if it's the target mail or the from mail.

Reply:
'send'

Reply:
done